src/daemon/rpmostreed-transaction-types.cxx

```diff
@@ -2582,6 +2598,10 @@ finalize_deployment_transaction_execute (RpmostreedTransaction *transaction,
   OstreeSysroot *sysroot = rpmostreed_transaction_get_sysroot (transaction);
   OstreeRepo *repo = ostree_sysroot_repo (sysroot);
 
+  // Check for inhibitor locks before unlocking staged deployment.
+  if (!check_sd_inhibitor_locks (cancellable, error))
```

Comment:
We're doing this twice in this function?

Reply:
ah oops. I had this for ease of manual testing and forgot to remove it :(